exercises/ex05-reminder/README.md

```diff
@@ -4,10 +4,11 @@ Substrate offers a way to automatically execute code on some events (when a bloc
 
```

Comment:
Add a sentence to explain that there is many types of hooks but we will focus here on on_initialize and on_finalize, that will give you a good understanding of how things works

Reply:
https://paritytech.github.io/substrate/master/frame_support/traits/trait.Hooks.html#method.on_idle
Add a link to this page of doc for the curious ones